vendor/github.com/moby/sys/mountinfo/LICENSE

```diff
@@ -1,5 +1,3 @@
-= vendor/go.opencensus.io licensed under: =
```

Comment:
git marked this as a move, which masked the fact that something in this PR resulted in this license file getting dropped from LICENSES.